backend/controller/ingress/request_test.go

```diff
@@ -84,79 +85,80 @@ func TestBuildRequestBody(t *testing.T) {
 	`)
 	assert.NoError(t, err)
 	for _, test := range []struct {
-		name     string
-		verb     string
-		method   string
-		path     string
-		query    url.Values
-		body     obj
-		expected any
-		err      string
+		name      string
+		verb      string
+		method    string
+		path      string
+		routePath string
+		query     url.Values
+		body      obj
+		expected  any
+		err       string
 	}{
 		{name: "UnknownVerb",
 			verb: "unknown",
 			err:  `unknown verb "unknown"`},
 		{name: "UnknownModule",
 			verb: "unknown",
 			err:  `unknown verb "unknown"`},
-		//FIXME: Query parameter decoding doesn't work?
-		//
-		// {name: "QueryParameterDecoding",
-		// 	verb:   "getAlias",
-		// 	method: "GET",
-		// 	path:   "/getAlias",
-		// 	query: map[string][]string{
-		// 		"alias": {"value"},
-		// 	},
-		// 	expected: HTTPRequest[AliasRequest]{
-		// 		Method: "GET",
-		// 		Path:   "/getAlias",
-		// 		Query: map[string][]string{
-		// 			"alias": {"value"},
-		// 		},
-		// 		Body: AliasRequest{
-		// 			Aliased: "value",
-		// 		},
-		// 	},
-		// },
+		{name: "QueryParameterDecoding",
+			verb:      "getAlias",
+			method:    "GET",
+			path:      "/getAlias",
+			routePath: "/getAlias",
+			query: map[string][]string{
+				"alias": {"value"},
+			},
+			expected: HTTPRequest[AliasRequest]{
+				Method: "GET",
+				Path:   "/getAlias",
+				Query: map[string][]string{
+					"alias": {"value"},
+				},
+				Body: AliasRequest{
+					Aliased: "value",
+				},
+			},
+		},
 		{name: "AllowMissingFieldTypes",
-			verb:   "postMissingTypes",
-			method: "POST",
-			path:   "/postMissingTypes",
+			verb:      "postMissingTypes",
+			method:    "POST",
+			path:      "/postMissingTypes",
+			routePath: "/postMissingTypes",
 			expected: HTTPRequest[MissingTypes]{
 				Method: "POST",
 				Path:   "/postMissingTypes",
 				Body:   MissingTypes{},
 			},
 		},
 		{name: "JSONPayload",
-			verb:   "postJsonPayload",
-			method: "POST",
-			path:   "/postJsonPayload",
-			body:   obj{"foo": "bar"},
+			verb:      "postJsonPayload",
+			method:    "POST",
+			path:      "/postJsonPayload",
+			routePath: "/postJsonPayload",
+			body:      obj{"foo": "bar"},
 			expected: HTTPRequest[PostJSONPayload]{
 				Method: "POST",
 				Path:   "/postJsonPayload",
 				Body:   PostJSONPayload{Foo: "bar"},
 			},
 		},
-		// FIXME: Path parameters don't seem to be interpolated into body fields?
-		//
-		// {name: "PathParameterDecoding",
-		// 	verb:  "getPath",
-		// 	method: "GET",
-		// 	path:   "/getPath/bob",
-		// 	expected: HTTPRequest[PathParameterRequest]{
-		// 		Method: "GET",
-		// 		Path:   "/getPath/bob",
-		// 		PathParameters: map[string]string{
-		// 			"username": "bob",
-		// 		},
-		// 		Body: PathParameterRequest{
-		// 			Username: "bob",
-		// 		},
-		// 	},
-		// },
+		{name: "PathParameterDecoding",
+			verb:      "getPath",
+			method:    "GET",
+			path:      "/getPath/bob",
+			routePath: "/getPath/{username}",
```

Comment:
For the path parameters, we need to specify the route as it would be in the route table so that we can pull out the fields in `BuildRequestBody`